burn-train/Cargo.toml

```diff
@@ -10,21 +10,31 @@ readme = "README.md"
 repository = "https://github.com/burn-rs/burn/tree/main/burn-train"
 version = "0.9.0"
 
+[features]
+cli = [
+    "indicatif",
+    "rgb",
+    "terminal_size",
+    "textplots",
+    "nvml-wrapper",
+    "sysinfo",
+    "systemstat"
```

Comment:
Those are used to compute metrics as well, you may still want them to be computed without the CLI to show them with a plot.

Probably not `nvml-wrapper`, since it's just text, but the other two can be useful regardless.

Reply:
Ok, I've split these off into a separate flag - does that work?